proto/p2p/p2p.proto

```diff
@@ -64,9 +64,14 @@ message Message {
 // Message that the local node sends to its remote peers,
 // in order to periodically check its uptime.
 //
+// Deprecated: remove this comment in the future.
 // On receiving "ping", the remote peer responds with the observed
 // uptime value of the message sender in "pong" message.
-message Ping {}
+message Ping {
+  // subnet_uptimes contains subnet uptime percentages, including the primary
+  // network.
```

Comment:
Can we separate out the primary network into its own field (like in pong) to save ~33 bytes per ping message

Reply:
done